### DISCUSSION TITLE
Crashes after a limited amount of runs

### DISCUSSION THREAD

**Discussion by draytalon** [FarbstoffRSL/RSL-Helper#367] [Q&A]
Firstly, A1 for the great app Farbstoff. Been using RSL for about a year now, but just recently have come across an issue of which i cannot resolve. I setup the helper to run overnight or during the day while i am asleep or away from the pc. So i go into a run and set it to 300 max runs, its a lot but that way i am guaranteed it to continue till i get back to the pc. Well when i do get back to the pc, the rsl helper has stopped at around 12 runs, sometimes i get to 16 but generally no more. This has only just happened within the last 2 weeks. I deleted all files and cache, redownloaded and installed, but the issue still persists. Was wondering if anyone else knows what causes it to stop after so many runs.

**Comment by YLemelin09** [comment]
Having the same issue on all auto-clicker options: food, dungeon auto-sell, mino

**Reply by quemazote** [reply]
I had this happen a couple of times last week. I thought it was how my selling rules were setup, but if others are seeing it, then it makes sense. Unfortunately we have a new RSL build today. Hopefully they can fix both issues.
